### PR TITLE
SyncTERM.SyncTERM version 1.1

### PR DESCRIPTION
**Pull request by markandpreston123**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/57469)

### CODE REVIEW

File: manifests/s/SyncTERM/SyncTERM/1.1/SyncTERM.SyncTERM.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: SyncTERM.SyncTERM
+PackageVersion: 1.1
+PackageLocale: en-US
+Publisher: SyncTERM
+PackageName: SyncTERM
+License: idk
```
Comment (Trenly) [inline manifests/s/SyncTERM/SyncTERM/1.1/SyncTERM.SyncTERM.locale.en-US.yaml:9]:
```suggestion
License: GPLv2
```

**Review by Trenly — CHANGES_REQUESTED** [review]
`idk` is not a valid license

Please update it with the information from their SourceForge page
image

File: manifests/s/SyncTERM/SyncTERM/1.1/SyncTERM.SyncTERM.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: SyncTERM.SyncTERM
+PackageVersion: 1.1
+PackageLocale: en-US
+Publisher: SyncTERM
+PackageName: SyncTERM
+License: GPLv2
```
Comment (ItzLevvie) [inline manifests/s/SyncTERM/SyncTERM/1.1/SyncTERM.SyncTERM.locale.en-US.yaml:9]:
```suggestion
License: GPL-2.0
```
to match SPDX.

File: manifests/s/SyncTERM/SyncTERM/1.1/SyncTERM.SyncTERM.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: SyncTERM.SyncTERM
+PackageVersion: 1.1
+Installers:
+- Architecture: x86
+  InstallerType: inno
+  InstallerUrl: https://netix.dl.sourceforge.net/project/syncterm/syncterm/syncterm-1.1/SyncTERM-1.1-setup.exe
```
Comment (ItzLevvie) [inline manifests/s/SyncTERM/SyncTERM/1.1/SyncTERM.SyncTERM.installer.yaml:9]:
```suggestion
  InstallerUrl: https://sourceforge.net/projects/syncterm/files/syncterm/syncterm-1.1/SyncTERM-1.1-setup.exe/download
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by lechacon** [comment]
The URL from sourceforge looks good. We can add the waiver for Unapproved-URL, but I'll wait as the URI may change as per @ItzLevvie 's suggestion.

**Pull request closed without merging**